### PR TITLE
Adding dynodao.eth

### PR DESCRIPTION
**Pull request by blockchainreg**
#### Hi! What is your PR about?

- [x] Add or edit a skin
- [x] Add a custom domain for your space
- [x] Add an alias to migrate your space

### CODE REVIEW

File: spaces/domains.json [hunk 1]
```diff
@@ -828,4 +828,5 @@
   "vote.wandinvestments.com": "wandinvestments.com",
   "vote.lebo.finance": "lebo-finance.eth",
   "vote.cirip.io": "cirip.io"
```
Comment (gokaiorg) [inline spaces/domains.json:830]: A comma is missing at the end of the line of your sub-domain.
```suggestion
  "vote.cirip.io": "cirip.io",
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Review by gokaiorg — CHANGES_REQUESTED** [review]
There is a conflict in your PR since the `domains.json` file has been updated, can you resolve it?
To prevent conflicts, it is recommended to add your subdomain between two existing domains rather than at the end or beginning of the list.

**Comment by blockchainreg** [comment]
hows everything looking

**Comment by gokaiorg** [comment]
> hows everything looking

Hey @blockchainreg The conflict still has to be resolved.

**Pull request closed without merging**